docs/src/model_providers_and_supported_functions.md

```diff
@@ -0,0 +1,30 @@
+```@meta
+CurrentModule = PromptingTools
+```
+
+# Model Providers and Supported Functions
+
+PromptingTools.jl routes AI calls through the use of subtypes of AbstractPromptSchema, which determine how data is formatted and where it is sent. (For example, OpenAI models have the corresponding subtype AbstractOpenAISchema, having the corresponding schemas - OpenAISchema, CustomOpenAISchema, etc.) This ensures that the data is correctly formatted for the specific AI model provider. 
+
+Below is an overview of the model providers supported by PromptingTools.jl, along with the corresponding schema information.
+
+| Abstract Schema         | Schema                    | Model Provider                         | aigenerate | aiembed | aiclassify | aiextract | aiscan | aiimage |
```

Comment:
If it's not too much effort, I think it would look visually much better to have aiextract be the 3rd column... Because it's much more ticks / more supported.

Unless it's way too much manual effort

Reply:
Just mark it as resolved when you're done.